haas/api.py

```diff
@@ -22,6 +22,7 @@
 from schema import Schema, Optional
 
 from haas import model
+from haas.auth import get_auth_backend
```

Comment:
I'm disinclined to mess with it until after the flask refactoring; flask has something akin to our `local`, but the db session has it's own request local outside of that, and there may be a better pattern to be had. `get_*()` gives us the ability to tweak exactly where the thing is stored without having to tear through the code changing everything.

These questions *will* come up during the flask integration, and we'll have to answer this for both the network allocator and the auth backend. Making both of these exposed request locals might be sensible, but let's hold off until the dust settles on that.